<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedMenuWidget</class>
 <widget class="QWidget" name="AdvancedMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BUTTON_AUTO_CONNECTION_OFF">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(30, 30, 30);
	color: rgb(70, 70, 70);
}</string>
   </property>
   <property name="text">
    <string>AUTO CONNECTION OFF</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUTO_PAIRING_ON">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(130, 130, 130);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>AUTO PAIRING ON</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUTO_CONNECTION_ON">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(130, 130, 130);
	color: rgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>AUTO CONNECTION ON</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTTON_AUTO_PAIRING_OFF">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: rgb(30, 30, 30);
	color: rgb(70, 70, 70);
}</string>
   </property>
   <property name="text">
    <string>AUTO PAIRING OFF</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>BUTTON_AUTO_CONNECTION_ON</zorder>
  <zorder>BUTTON_AUTO_PAIRING_OFF</zorder>
  <zorder>BUTTON_AUTO_CONNECTION_OFF</zorder>
  <zorder>BUTTON_AUTO_PAIRING_ON</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
